.eslintrc.js

```diff
@@ -7,7 +9,7 @@
         "@typescript-eslint"
     ],
     "parserOptions": {
-        "project": "./tsconfig.lint.json"
+        "project": path.join(__dirname, "tsconfig.lint.json"),
```

Comment:
Just picking up some rocks.. the configuration was valid but caused an error in VSCode, preventing the eslint extension from working correctly.